packages/bpk-component-rating/src/BpkRating.js

```diff
@@ -132,7 +132,7 @@ BpkRating.propTypes = {
 
 BpkRating.defaultProps = {
   className: null,
-  size: RATING_SIZES.base,
+  size: 'sm',
```

Comment:
Does this need to change? The Rating Size is not the same as the text size so we can keep the current interface while changing the text styles used?

Reply:
Yes because of the font sizes needed to change but I wanted to make sure that it didn't break the existing API for users, otherwise it meant adding additional JS logic to this file to accomplish the same result